go.mod

```diff
@@ -1,5 +1,6 @@
 module github.com/notional-labs/composable/v6
 
+// TODO: move module to github.com/ComposableFi/composable-cosmos
```

Comment:
we all know it to be moved ;) so usually i do not create such todo - we all know .

Reply:
removed at https://github.com/ComposableFi/composable-cosmos/pull/408/commits/fb2e92d01fa0868e8de9d7c064b27c5224dc1513